maintenance/scripts/delete_rendering.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# This script deletes rnd settings on images in specified Datasets.
+# Datasets can be specified either by name, such as
+# DATASETNAME=svs bash delete_rendering.sh
+# or by IDs (one or more, separated by comma), such as
+# DATASETID=1868,2073 bash delete_rendering.sh
+
+echo Starting
+OMEROPATH=${OMEROPATH:-/opt/omero/server/OMERO.server/bin/omero}
+PASSWORD=${PASSWORD:-ome}
+HOST=${HOST:-outreach.openmicroscopy.org}
+DATASETNAME=${DATASETNAME:-siRNAi-HeLa}
+DATASETIDS=${DATASETID:-none}
+NUMBER=${NUMBER:-50}
+OMEUSER=${OMEUSER:-trainer-1}
+
+$OMEROPATH login -u $OMEUSER -s $HOST -w $PASSWORD
+result=`$OMEROPATH hql --ids-only --limit 1000 --style plain -q --all "SELECT id from Dataset WHERE name = '$DATASETNAME'"`
```

Comment:
Should this be inside the `then`?

Reply:
It should, thanks you. Fixed by https://github.com/ome/training-scripts/pull/58/commits/a57e0745675420a7c7b3b16489e655630000d9c1